docs/tools/CLI/cli-compile.md

```diff
@@ -147,8 +168,8 @@ $ mbed compile --config -t GCC_ARM -m K64F --prefix target --prefix app
 
 ### Define macros
 
-Define macros when compiling by using the `-D` argument:
+To define macros when compiling, use the `-D` argument:<!--are you defining them here, or calling them? and is this the same as https://os.mbed.com/docs/mbed-os/v5.12/reference/configuration.html-->
```

Comment:
The proper terminology would be you are defining them, not calling them. It is related to the macros on that page, however there is not dependency on the macros being specified in JSON files beforehand.